lib/utils/isValidIdentifier.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
```

Comment:
Should this be its own file? And hopefully, I haven't missed an existing version of this util!